Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12-slim
+LABEL maintainer="m.shmygel@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
```

Comment:
The `COPY requirements.txt requirements.txt` command is fine, but ensure that your `requirements.txt` file is present in the context directory. This is important for the subsequent `RUN pip install -r requirements.txt` command to work correctly.